src/containers/ServiceList/ServiceList.test.jsx

```diff
@@ -0,0 +1,73 @@
+import { mount, configure } from 'enzyme';
+import React from 'react';
+import ServiceList from './ServiceList';
+import Adapter from 'enzyme-adapter-react-16';
+import { Card } from '@material-ui/core';
+
+configure({adapter: new Adapter()});
+
+describe('<ServiceList />', () => {
+	const service = {
+		name: 'test',
+		id: 1,
+		tutorInfo: {
+			user: {
+				firstName: 'testtur',
+				lastName: 'testtursen'
+			}
+		}
+	};
+	const services = Array(5).fill(service);
+	let wrapper;
+	beforeEach(() => {
+		wrapper = mount(<ServiceList />);
```

Comment:
Hvorfor mount i stedet for shallow?

Reply:
Fordi at den bruger componenter fra material-ui som er en masse nestede components. Hvis jeg bruger shallow vil de ikke blive rendered